travis/check-image-icc-profiles.sh

```diff
@@ -15,6 +15,6 @@ TRAVIS_PHP_VERSION=$2
 
 if [ "$TRAVIS_PHP_VERSION" == "5.3.3" -a "$DB" == "mysqli" ]
 then
-	find . -type f -not -path './phpBB/vendor/*' -iregex '.*\.\(gif\|jpg\|jpeg\|png\)$' | \
+	find . -type f -not -path './phpBB/*' -iregex '(vendor\|ext/vendor/).*\.\(gif\|jpg\|jpeg\|png\)$' | \
```

Comment:
you are ignoring all the files inside phpBB.

`find . -type f -a -iregex '.*\.\(gif\|jpg\|jpeg\|png\)$' -a -not -wholename '*vendor/*'` should work